### DISCUSSION TITLE
PyWebView application architecture

### DISCUSSION THREAD

**Discussion by swampy747** [r0x0r/pywebview#791] [Ideas]
Does anyone have a recommended way to architect applications with PyWebView?  I'm use to MVC or MVVM but I really have not successfully organized my code well to work with PyWebView.  That is to have a class/file per HTML page for example.

Does anyone have any good examples they'd like to share?

**Comment by r0x0r** [comment]
I made this app with Angular and pywebview a while back https://github.com/r0x0r/traktorlibrarian
It is somewhat outdated, but may provide some guidance on how a project can be structured.
